### ISSUE TITLE
!votemap isn't working all of a sudden?

### ISSUE THREAD

**Issue by ThomasAdnum** [abnerfs/cs2-rockthevote#68]
Hello!
!votemap was working fine, I enable RTV just for fun and then once I disabled RTV now !votemap doesn't work.

Any help would be appreciated. 

`// This configuration was automatically generated by CounterStrikeSharp for plugin 'RockTheVote', at 2024/04/22 11:17:03
{
  "Version": 9,
  "Rtv": {
    "Enabled": false,
    "EnabledInWarmup": false,
    "NominationEnabled": false,
    "MinPlayers": 0,
    "MinRounds": 0,
    "ChangeMapImmediatly": true,
    "HideHudAfterVote": false,
    "MapsToShow": 24,
    "VoteDuration": 30,
    "VotePercentage": 60,
    "HudMenu": true
  },
  "Votemap": {
    "Enabled": true,
    "VotePercentage": 60,
    "ChangeMapImmediatly": true,
    "EnabledInWarmup": true,
    "MinPlayers": 0,
    "MinRounds": 0,
    "HudMenu": true
  },
  "EndOfMapVote": {
    "Enabled": true,
    "MapsToShow": 6,
    "HudMenu": true,
    "ChangeMapImmediatly": false,
    "VoteDuration": 30,
    "HideHudAfterVote": false,
    "TriggerSecondsBeforeEnd": 120,
    "TriggerRoundsBeforEnd": 2,
    "DelayToChangeInTheEnd": 6
  },
  "Timeleft": {
    "ShowToAll": false
  },
  "Nextmap": {
    "ShowToAll": false
  },
  "MapsInCoolDown": 0
}`

All I changed was RTV to TRUE, and then I set it back to false and rebooted the server. I dont know why !votemap wouldn't be working.

**Comment by b1gwest** [comment]
Same issue with votemap not working. I deleted the config file and rebooted server to generate new config file. Now 'votemap' does not show list of maps to choose from. Doing votemap <map> still works

**Comment by ThomasAdnum** [comment]
> Same issue with votemap not working. I deleted the config file and rebooted server to generate new config file. Now 'votemap' does not show list of maps to choose from. Doing votemap still works

So deleting the config file fixed it? I'll give it a go when I get home.

**Comment by b1gwest** [comment]
No, it was working Ok before I deleted the config file. Now I cant get votemap working with the new generated config
